tortoisebeacon/weakcoin/weak_coin.go

```diff
@@ -290,19 +290,17 @@ func (wc *WeakCoin) FinishRound() {
 	coinflip := wc.smallestProposal[lsbIndex]&1 == 1
 
 	wc.coins[wc.round] = coinflip
-	wc.logger.With().Info("completed round",
-		log.Uint32("epoch_id", uint32(wc.epoch)),
-		log.Uint32("round_id", uint32(wc.round)),
+	logger.With().Info("completed round",
 		log.String("proposal", types.BytesToHash(wc.smallestProposal).ShortString()),
-		log.Bool("coinflip", coinflip))
+		log.Bool("tb_weakcoin", coinflip))
```

Comment:
whats the point of tb prefix?

Reply:
i was trying to differentiate between self-healing weakcoin and tortoise beacon weakcoin